# HG changeset patch
# User Rik <rik@octave.org>
# Date 1541828324 28800
#      Fri Nov 09 21:38:44 2018 -0800
# Node ID 1357b79f87bc09c85902469b8110435f7e39523d
# Parent  ff0aa19333c942ca1fe2614f8139f463393399be
doc: grammar check txi files in doc/interpreter/ directory.

* external.txi, install.txi: grammar check txi files.

diff --git a/doc/interpreter/external.txi b/doc/interpreter/external.txi
--- a/doc/interpreter/external.txi
+++ b/doc/interpreter/external.txi
@@ -2096,19 +2096,19 @@ javaObject ("java.math.BigDecimal", "1")
 
 In order to execute Java code Octave creates a Java Virtual Machine (JVM).  By
 default the version of the JVM is used that was detected during configuration
 on Unix-like systems or that is pointed to from the registry keys at
 @file{HKEY_LOCAL_MACHINE\SOFTWARE\JavaSoft\JRE} or
 @file{HKEY_LOCAL_MACHINE\SOFTWARE\JavaSoft\Java Runtime Environment} on
 Windows.  The default path to the JVM can be overwritten by setting the
 environment variable @w{@env{JAVA_HOME}} to the path where the JVM is
-installed.  On Windows that might be e.g. @file{C:\Program
-Files\Java\jre-10.0.2}.  Make sure that you select a directory that contains
-the JVM with a bit-ness that matches Octave's.
+installed.  On Windows that might be, for example,
+@file{C:\Program Files\Java\jre-10.0.2}.  Make sure that you select a directory
+that contains the JVM with a bit-ness that matches Octave's.
 
 The JVM is only loaded once per Octave session.  Thus, to change the used
 version of the JVM, you might have to re-start Octave.  To check which version
 of the JVM is currently being used, run @code{version -java}.
 
 The JVM allocates a fixed amount of initial memory and may expand this pool up
 to a fixed maximum memory limit.  The default values depend on the Java version
 (@pxref{XREFjavamem,,javamem}).  The memory pool is shared by all Java objects
diff --git a/doc/interpreter/install.txi b/doc/interpreter/install.txi
--- a/doc/interpreter/install.txi
+++ b/doc/interpreter/install.txi
@@ -308,17 +308,17 @@ provide improved performance for the fun
 @item QScintilla
 Source code highlighter and manipulator; a Qt port of Scintilla
 (@url{http://www.riverbankcomputing.co.uk/software/qscintilla}).
 QScintilla is used for syntax highlighting and code completion in the
 GUI.
 
 @item Qt
 GUI and utility libraries (@url{https://www.qt.io}).  Qt is required for
-building the GUI.  It is a large framework, but the only components
+building the GUI@.  It is a large framework, but the only components
 required are the GUI, core, and network modules.
 
 @item SuiteSparse
 Sparse matrix factorization library
 (@url{http://faculty.cse.tamu.edu/davis/suitesparse.html}).
 SuiteSparse is required to provide sparse matrix factorizations and
 solution of linear equations for sparse systems.
 
